themes/default/content/blog/announcing-public-preview-update-plans/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Announcing the public preview of Update Plans"
+date: 2022-02-10T08:00:00-07:00 # TODO update this
+meta_desc: Update Plans enable you to see and confirm the updates that will be made to your infrastructure and then apply those updates at a later time.
+meta_image: meta.png # TODO update this
+authors:
+    - alex-mullans
```

Comment:
```suggestion
```

Reply:
I wasn't sure about this. He has left, but he did write most of this and I kinda felt credit due where credit's due?